dev-utils/docker-compose.yml

```diff
@@ -2,14 +2,17 @@ version: "2.1"
 services:
   elasticsearch:
     container_name: elasticsearch
-    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION:-7.17.0}
+    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION:-8.1.2}
     environment:
       - cluster.name=docker-cluster
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-XX:UseAVX=2 -Xms1g -Xmx1g"
       - "cluster.routing.allocation.disk.threshold_enabled=false"
       - "discovery.type=single-node"
       - "path.data=/usr/share/elasticsearch/data/${STACK_VERSION}"
+      - "xpack.security.http.ssl.enabled=false"
```

Comment:
Done because of the sourcemaps test:  https://github.com/elastic/apm-agent-rum-js/blob/main/packages/rum/test/integration/upload-sourcemaps.spec.js